### PR TITLE
Issue #4136: Split and Organize Checkstyle inputs by Test for CatchParameterName

### PR DESCRIPTION
**Pull request by Kietzmann**
#4136

### CODE REVIEW

File: src/test/resources/com/puppycrawl/tools/checkstyle/checks/naming/catchparametername/InputCatchParameterNameSimple.java [hunk 1]
```diff
@@ -0,0 +1,225 @@
...
+import java.io.*;
+/**
+ * Contains simple mistakes:
+ * - Long lines
+ * - Tabs
+ * - Format of variables and parameters
+ * - Order of modifiers
+ * @author Oliver Burn
+ **/
+final class InputCatchParameterSimple
```
Comment (rnveach) [inline src/test/resources/com/puppycrawl/tools/checkstyle/checks/naming/catchparametername/InputCatchParameterNameSimple.java]: Please rename class to same as file, `InputCatchParameterNameSimple`.

Comment (Kietzmann) [inline src/test/resources/com/puppycrawl/tools/checkstyle/checks/naming/catchparametername/InputCatchParameterNameSimple.java]: Done

**Comment by romani** [comment]
CIs should be green, please rebase on our latest master to resolve TC issue

**Comment by Kietzmann** [comment]
Done

**Comment by romani** [comment]
please rebase on our latest master to fix werker. TC failure due to license could be ignored, I reported and issue to their support.

**Comment by Kietzmann** [comment]
@romani 
I made rebase on recent changes in the main master, but wercker still failing(. Could you please provide some logs for wercker, cause I don't have access to build details.
Thanks

**Comment by rnveach** [comment]
We cannot give out read-only access to wercker, it is an issue with them.
wercker error is the silly `Could not resolve host: github.com`.

**Pull request merged**